### ISSUE TITLE
[text-field] Automatic handling of the `aria-invalid` attrinute

### ISSUE THREAD

**Issue by Ross-Blakeney** [tyler-technologies-oss/forge#1009]
**Describe the feature:**
Currently, the `aria-invalid` attribute is not handled automatically by Forge. It should be in order to ease a11y efforts on adopting teams.

**Describe any alternatives you've considered:**
Always handle the `aria-invalid` attribute on the consumer's side. Impractical by comparison to handling this in the component itself.

**Additional context:**
This PR from the a11y team brought this to my attention.
https://github.com/tyler-technologies/tcp-admincenter/pull/1626/files#r2473507826

Image showing an invalid text-field not having the `aria-invalid` attribute added as expected.

<img width="2398" height="289" alt="Image" src="https://github.com/user-attachments/assets/477e96cb-2171-454a-9200-4a723f490359" />
